packages/compass-indexes/src/components/drop-index-modal/drop-index-modal.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
nit: do we need this file? actually name of the file is a bit off. modal is in `drop-index-form` file. (feel free to ignore)